frontend/app/templates/components/admin-component.hbs

```diff
@@ -0,0 +1,4 @@
+{{#if session.isAuthenticated}}
+    <h1>Admin</h1>
+{{/if}}
+{{yield}}
```

Comment:
There are no elements to be passed in the component (DOM elements) so no need of yield, or there is any specific use case here? All the components will be displayed except the heading admin. Do you think it is relevant to use such thing?

Reply:
If I remove it than `ember test` fails

Reply:
It should not be the case. Please try to look in the test cases as our tests are passing. Please look into the issue